php/app/Http/Middleware/TrustProxies.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Illuminate\Http\Middleware\TrustProxies as Middleware;
+use Illuminate\Http\Request;
+
+class TrustProxies extends Middleware
+{
+    /**
+     * The trusted proxies for this application.
+     *
+     * @var array<int, string>|string|null
+     */
+    protected $proxies;
+
+    /**
```

Comment:
Is this pretty much setting up cors? This is not an issue, but just curious.

Reply:
This is some laravel stuff that comes out of the box. tbh its beyond my knowledge as of today.